### PR TITLE
KTX2 - Fixing context issue and vis tests

### PR DESCRIPTION
**Pull request by RaananW**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14825/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14825/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14825/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14825/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14825/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14825/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14825/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14825/merge#BCU1XR#0

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14825/merge/testResults/webgl2playwright/index.html

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)
Important - these might fail sporadically. This is an optional test.

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14825/merge/testResults/webgpuplaywright/index.html

File: packages/tools/tests/test/visualization/config.json [hunk 1]
```diff
@@ -1116,7 +1116,7 @@
             "playgroundId": "#EIJH8L#27",
             "renderCount": 5,
             "referenceImage": "ktx2decoder.png",
-            "excludeFromAutomaticTesting": true
+            "excludedEngines": ["webgl1", "webgpu"]
```
Comment (RaananW) [inline packages/tools/tests/test/visualization/config.json:1119]:
```suggestion
            "excludeFromAutomaticTesting": true
```

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14825/merge/testResults/webgl2playwright/index.html

**Review by Popov72 — APPROVED** [review]

**Pull request merged**